docs/experimentsetup.rst

```diff
@@ -3,27 +3,98 @@
 Experiment Setup
 =================
 
-Three-electrode experiment
----------------------------
+A variety of cells can be characterized using a potentiostat. This section will briefly describe cell preparation and
+attachment to the potentiostat.
+
+Types of Cell Arrangements
+--------------------------
+
+Most potentiostats (including the JUAMI Potentiostat) have the option to connect three leads to an electrochemical cell.
+Electrochemical cells/devices commonly have two or three leads. An important distinction between the two arrangements is the
+dependencies of the potential.
+
+In a three electrode cell, there is a working electrode (with the material or reaction of interest) and counter electrode
+(completes the circuit and allows charge to flow by facilitating a counter reaction) and a reference electrode.
+The reference electrode does not undergo any reactions, and it maintains a constant potential. The standard hydrogen
+electrode (SHE) is based on the formation of hydrogen gas from two protons and two electrons on a platinum electrode in an ideal solution.
+The potential at which this reaction occurs is defined as 0 Volts on the scale of reduction potentials, from which other
+reactions can be referenced.
```

Comment:
I'm not sure if it's ideal to describe the hydrogen electrode in the experiment setup. They will probably not be using a hydrogen electrode. Most likely they'll use AgCl or calomel electrode and that's what will determine the voltage that they read. I think instead, its should say that users will get a voltage reading that is dependent on what type of reference they use in the experiment.

Reply:
Yeah that's probably best. We can avoid specifics here since this is a general getting started page.

For fullness maybe there can be a section on reference electrodes later that this links to where the required level of detail is appropriate.

Reply:
Both good suggestions. I anticipate adding a section that gives background on general echem and the specific techniques for people that have never really done echem experiments. I think it could fit into that section, but it is not necessary for this release.

Reply:
@jlhitt1993 @aplymill7 both comments resolved.

Reply:
It looks good to me. So did we decide that we are just going to build the HTML on the user's computer? I can't remember it that happens automatically in setup.py or not.

Reply:
I just checked and it doesn't look like it does. That means you need to build with sphinx and then commit the HTML.

Reply:
The _build folder is part of the gitignore file, which @sbillinge mentioned was the right way of doing things. If I remember the conversation correctly, the setup.py file should build the html. To do so, we will also need to pip install sphinx_rtd_theme

Reply:
This might be helpful: https://www.sphinx-doc.org/en/master/usage/advanced/setuptools.html

Reply:
I don't think our setup does the build as of now but if you look in the docs folder, there are already HTML files.

Reply:
The docs\index.html file is outdated and should be removed, and the docs\html folder is also outdated. I think these may have snuck through in an early PR where the sphinx config wasn't formatted properly. I think it would be good to create two issues once this is pulled in. First clean up the docs folder (by removing these outdated files and folders). Second to add building docs in setup.py.